### PR TITLE
Feature/103 Implemented Account Receivable for backend

### LINKED ISSUE #103: Implement Account Receivable backend

### PR DESCRIPTION
**Pull request by nasaku898**
resolves #103 

Get All Account Receivable based on user token

Side note, remember to wipe your local database since I have change certain definition

image

Get Bikes based on account receivable

image

### CODE REVIEW

File: bike-erp/backend/src/main/dao/AccountReceivableDAO.ts [hunk 1]
```diff
@@ -0,0 +1,59 @@
+import db from "../helpers/db"
+
```
Comment (jmezaber) [inline bike-erp/backend/src/main/dao/AccountReceivableDAO.ts:2]: Don't forget to put comments!

**Review by jmezaber — CHANGES_REQUESTED** [review]
Looks good! You are missing a couple of semi-column and don't forget to add comments!

File: bike-erp/backend/src/main/dao/AccountReceivableDAO.ts (hunk 1 above)
Comment (nasaku898) [reply bike-erp/backend/src/main/dao/AccountReceivableDAO.ts:2]: Comment added!

**Review by Manraj45 — APPROVED** [review]
Good job!

**Review by jmezaber — APPROVED** [review]
Amazing job as always!

**Pull request merged**